data_integration/logging/chat_room.py

```diff
@@ -0,0 +1,58 @@
+import abc
+from data_integration.logging import events
+
+
+class ChatRoom(abc.ABC):
+
+    def __init__(self, chat_type, code_markup_start: str, code_markup_end: str, line_start: str,
+                 line_end: str = '', replace_with: str = '_'):
+        self.chat_type = chat_type
+        self.code_markup_start = code_markup_start
+        self.code_markup_end = code_markup_end
+        self.line_start = line_start
+        self.line_end = line_end
+        self.replace_with = replace_with
+        self.line_end = line_end
+
+    @abc.abstractmethod
+    def create_error_text(self, node_path: []):
+        pass
+
+    @abc.abstractmethod
+    def create_error_msg(self, text, error_log1, error_log2):

```

Comment:
`error_logs: []` (list instead of multiple parameter)?

Reply:
Done